tests/test_matrix.py

```diff
@@ -104,7 +109,14 @@ def test_none_assignment_and_merging_in_dict(
             node = node_type(value=value, is_optional=False)
             data = {"node": node}
             cfg = OmegaConf.create(obj=data)
-            verify(cfg, "node", none=False, opt=False, missing=False, inter=False)
+            verify(
+                cfg,
+                "node",
+                none_internal=False,
```

Comment:
can you rename non_internal back to none and undo the formatting changes when not needed (by removing the trailing comma)?

Reply:
Sure -- done in 44cd8af